Dockerfile

```diff
@@ -66,7 +69,7 @@ RUN curl -L https://www.kernel.org/pub/linux/utils/boot/syslinux/syslinux-6.03.t
 RUN ln -s /toolchain/bin/pwd /bin/pwd && \
     make installer && \
     cp /tmp/syslinux/bios/extlinux/extlinux /rootfs/bin && \
-    cp /tmp/syslinux/bios/mbr/gptmbr.bin /rootfs/share
+    cp /tmp/syslinux/efi64/mbr/gptmbr.bin /rootfs/share
```

Comment:
Wonder how many of these we need to support. Is efi64 enough to cover most/all use cases or do we need to copy over others?